internal/helm/plan.go

```diff
@@ -116,6 +116,10 @@ var upgrade = func(cfg env.Config) []Step {
 		steps = append(steps, run.NewDepUpdate(cfg))
 	}
 
+	if cfg.Lint {
+		steps = append(steps, lint(cfg)...)
```

Comment:
```suggestion
		steps = append(steps, run.NewLint(cfg))
```

Reply:
you can think about functions in this file as pipeline building functions, each one returns `[]Step` that is basically what steps to run. The `run.New*` constructors is to return a type associated to the helm cli subcommand.
You could use the `lint` function here but that's not the right approach as you will be appending steps that maybe are already done in upgrade or maybe modify the upgrade behavior. The correct approach is to use the Lint "object"

Reply:
I see thanks for clarifying that 👍